proto/src/chrono.rs

```diff
@@ -1,53 +0,0 @@
-use core::convert::TryInto;
-
-use chrono::{DateTime, Duration, TimeZone, Utc};
-
-use crate::google::protobuf as pb;
-
-impl From<DateTime<Utc>> for pb::Timestamp {
-    fn from(dt: DateTime<Utc>) -> pb::Timestamp {
-        pb::Timestamp {
-            seconds: dt.timestamp(),
-            // This can exceed 1_000_000_000 in the case of a leap second, but
```

Comment:
This was against the wire protocol [specification](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#timestamp).